src/elements/content-sidebar/SidebarUtils.js

```diff
@@ -119,6 +131,74 @@ class SidebarUtils {
                 SidebarUtils.shouldRenderMetadataSidebar(props, editors))
         );
     }
+
+    /**
+     * Gets the title for a given sidebar view
+     *
+     * @param {string} view - the view name
+     * @return {React.Node} - the node to render
+     */
+    static getTitleForView(view: SidebarView): React.Node {
+        switch (view) {
+            case SIDEBAR_VIEW_SKILLS:
+                return <FormattedMessage {...messages.sidebarSkillsTitle} />;
+            case SIDEBAR_VIEW_DETAILS:
+                return <FormattedMessage {...messages.sidebarDetailsTitle} />;
+            case SIDEBAR_VIEW_METADATA:
+                return <FormattedMessage {...messages.sidebarMetadataTitle} />;
+            case SIDEBAR_VIEW_ACTIVITY:
+                return <FormattedMessage {...messages.sidebarActivityTitle} />;
+            default:
+                return null;
+        }
+    }
+
+    /**
+     * Marks and gets the loader for a given sidebar view
+     *
+     * @param {String} view - the view name
+     * @param {String} markName -  the name to be used by performance.mark
+     * @return {Function} - a function which will resolve the module to load
+     */
+    static getLoaderForView(view: SidebarView, markName: string): Promise<any> {
```

Comment:
Is it possible to have this function in its own file + same additional stuff for `content-sidebar` and `content-open-with` also in it which are currently inside ContentPreview.js and Header.js.

So that all these are in 1 place/1 file to reference?

Reply:
The logic here is unique for sidebar, because we are creating a start mark. The start mark for preview and sidebar are created externally